### PR TITLE
19 navbar

### PR DESCRIPTION
**Pull request by bekk1029**
navbar mui icons

### CODE REVIEW

File: frontend/src/components/header/Search.tsx
Comment (Zorurunn) [inline frontend/src/components/header/Search.tsx:1]: ustgaarai

File: frontend/src/constants/index.ts
Comment (Zorurunn) [inline frontend/src/constants/index.ts:1]: ustgaarai

**Review by Zorurunn — CHANGES_REQUESTED** [review]

**Review by Zorurunn — APPROVED** [review]

**Review by ulaaka211 — APPROVED** [review]
za yahudee

**Review by Zorurunn — APPROVED** [review]

**Review by Zorurunn — CHANGES_REQUESTED** [review]
navbar aa 2 branchaar push hiisen ymuu ?
ali neg branhchiig ni songood merge hiihgui bol 
main ruu navbar iig 2ynzaar zasaad merge hiih geheer conflict uuseh bh te 
2 branchiin ali zow be ?
hereggui pull reqyuestee ustagchihgui yu doo

**Comment by Zorurunn** [comment]
aan ene nuguu conflict tei ni bn te eniigee ustagchihguiyu doo

**Pull request closed without merging**